### ISSUE TITLE
Q About BART bart-large-cnn

### ISSUE THREAD

**Issue by epsilon-deltta** [huggingface/transformers#13575]
I'm using this BART tokenizer code
```python
from transformers import BartForConditionalGeneration, BartTokenizer
model_name = "facebook/bart-large-cnn"
tokenizer = BartTokenizer.from_pretrained(model_name)
```
I wanna read the explanation of bart-large-cnn. 
I guess this BPE tokenization (the paper said it uses BPE same as GPT-2) is trained on CNN/DM (Hermann et al., 2015). is that right?
Q1. Which dataset did the above tokenization use?
Q2 If there is a documentation about 'bart-large-cnn, bart-large-mnli, bart-large-xsum, bart-eli5, and bart-base', could you leave the link.

Thanks in advance.

**Comment by patil-suraj** [comment]
Hi there!

Thank you for opening the issue. Please use the [forum](https://discuss.huggingface.co/)  for such general questions. Issues are used to report bugs or feature requests.

**Comment by epsilon-deltta** [comment]
So sorry I didn't know the forum exists
